Some misc fixes and cleanups in preparation for notifications

pjeweb opened the pull request:
just a few commits that can be merged separately

MattIPv4 commented on apps/website/src/components/content/Meta.tsx:
```diff
@@ -12,7 +12,7 @@ type MetaProps = {
 };
 
 // Get our base URL, which will either be specifically set, or from Vercel for preview deployments
-const BASE_URL = env.NEXT_PUBLIC_BASE_URL.replace(/\/$/, "");
```
Ditto

pjeweb commented on apps/website/build-scripts/create-manifest.ts:
```diff
@@ -19,7 +19,7 @@ const manifest = {
   ],
   theme_color: "#636a60",
   background_color: "#636a60",
-  start_url: `${env.NEXT_PUBLIC_BASE_URL.replace(/\/$/, "")}/homescreen`,
```
oh yeah the commit checking the trailing slash centrally is missing, oops

MattIPv4 replied:
Oh nvm I see it in the env schema -- I guess that'll just mean we throw an error instead of silently trimming it?

pjeweb replied:
no it was there, just the wrong order of commits: https://github.com/alveusgg/alveusgg/pull/237/files#diff-446483ae87e65c5118545af203aa7593bca1ebe598e5c875f3326feb57bbdba4R54

pjeweb replied:
does that work for you? it checks there is not trailing slash when parsing env vars during build

MattIPv4 replied:
Yeh works for me, just missed it when I was reading through, my bad

MattIPv4 commented:
👍 I had looked at `tsx` before, but I think I'd stuck with pure `esbuild` so that we could do custom loaders to stub out the image imports etc. -- that's no longer a concern though as I split those out into a dedicated file.

Does `esbuild` need to be a project dependency still? Are we using it elsewhere?

MattIPv4 commented on apps/website/src/components/shared/form/TextAreaField.tsx:
```diff
@@ -25,7 +25,7 @@ export function TextAreaField(props: TextAreaFieldProps) {
       <TextareaAutosize
         className={`h-[calc(1em+0.5rem)] max-h-[500px] min-h-[calc(1em+0.5rem)] w-full resize-y rounded-sm border border-gray-700 bg-white p-1 px-2 text-black ${
           props.inputClassName || ""
-        }`}
+        } text-black`}
```
text-black is here twice now?

pjeweb replied:
fixed

pjeweb commented:
> 👍 I had looked at `tsx` before, but I think I'd stuck with pure `esbuild` so that we could do custom loaders to stub out the image imports etc. -- that's no longer a concern though as I split those out into a dedicated file.
> 
> Does `esbuild` need to be a project dependency still? Are we using it elsewhere?

nope. removed it as dep
